src/main.rs

```diff
@@ -51,15 +51,15 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         &wgpu::SwapChainDescriptor {
             usage: wgpu::TextureUsage::OUTPUT_ATTACHMENT,
             format: render_format,
-            width: size.width.round() as u32,
-            height: size.height.round() as u32,
+            width: size.width,
+            height: size.height,
             present_mode: wgpu::PresentMode::Vsync,
         },
     );
 
     // TODO: Dynamically load fonts or something?
-    let inconsolata: &[u8] =
-        include_bytes!("/Users/connor/Library/Fonts/InconsolataGo-Regular.ttf");
+    let inconsolata: &[u8] = include_bytes!("/usr/share/fonts/truetype/ubuntu/UbuntuMono-R.ttf");
```

Comment:
We should probably just ship a font like Inconsolata in the repo and have a relative path here. (we would just need to make sure to add the font's license to the repo as well).

Reply:
How about https://github.com/servo/font-kit?

Reply:
Yeah it'd be great to use something like font-kit for this!